### ISSUE TITLE
Is it possible that you've made a mistake about the version of Pytorch.

### ISSUE THREAD

**Issue by marsggbo** [sg-nm/cgp-cnn-PyTorch#1]
```
 >>> pip install torch==0.2.0_4 torchvision
Collecting torch==0.2.0_4
  Could not find a version that satisfies the requirement torch==0.2.0_4 (from versions: 0.1.2, 0.1.2.post1, 0.3.1, 0.4.0, 0.4.1)
No matching distribution found for torch==0.2.0_4
```

**Comment by sg-nm** [comment]
Thank you for your post.
Currently, the version 0.2 seems to be invalid...
Can you please install another version of PyTorch (e.g, 0.4.1)?

**Comment by marsggbo** [comment]
![image.png](https://ask.qcloudimg.com/draft/1215004/j0i0vglwst.png)
![image.png](https://ask.qcloudimg.com/draft/1215004/tsz9yprd7d.png)

The above is the result obtained by 'python exp_main.py -g 2', but it seems not work. Moreover, though there are error reports, the code can still be run.

The environment are as following:
- CentOS 7
- Cuda 8.0
- Python 3.6
- Pytorch 0.4.1

**Comment by sg-nm** [comment]
Please change the code of line 199 and 267 in cnn_train.py file as follows:

`correct += predicted.eq(label_.data).cpu().sum().item()`

**Comment by marsggbo** [comment]
Thank you so much, the code succeded to run but with some future version warning.  Besides, I still have some questions:
1. The code has run for many many times, what the expected result should be? 
![](https://ask.qcloudimg.com/draft/1215004/4sr091e1pd.png)
2. Which part in the code I need to change if I want to transfer to my own dataset?

**Comment by sg-nm** [comment]
1. To get the results reported in our paper, you first need to perform the evolution process for 500 generations. Then you need to retrain the best architecture for 500 epochs using the retraining settings described in our paper.

2. In our code, we define our dataloader in the line 121 of cnn_train.py. For using your own dataset, you need to delete the line 121 in cnn_train.py and define your dataloader here.

**Comment by 1810325** [comment]
@sg-nm Why do I always download the CIFAR data set when I run the code, even though I've already downloaded it And if I want to port your code to my own data set, do I have to change my data set to CIFAR format, and my data is 32 * 420

**Comment by 1810325** [comment]
@marsggbo Have you ported the code to your own data set, can you teach me, I am a novice, at present very confused. . Thank you very much

**Comment by 1810325** [comment]
@sg-nm Hello, as the former said, my code has been running for a long time, why is the accuracy always high and low, and I must have run 500 rounds, when should I stop?

**Comment by 1810325** [comment]
@sg-nm Hello, as the former said, my code has been running for a long time, why is the accuracy always high and low, and I must have run 500 rounds, when should I stop?
![image](https://user-images.githubusercontent.com/47933430/68925229-8a604600-07bd-11ea-81b2-3fc70ce2316c.png)

**Comment by yuxiaomu** [comment]
> 非常感谢您，代码成功运行，但带有一些将来的版本警告。此外，我还有一些问题：
> 
> 1. 该代码已经运行了很多次，预期的结果应该是什么？
>    ![](https://camo.githubusercontent.com/1b4b440d164f6a373f961fb208f538b6987804b3/68747470733a2f2f61736b2e71636c6f7564696d672e636f6d2f64726166742f313231353030342f347372303931653170642e706e67)
> 2. 如果要转移到自己的数据集，我需要更改代码的哪一部分？


> 非常感谢您，代码成功运行，但带有一些将来的版本警告。此外，我还有一些问题：
> 
> 1. 该代码已经运行了很多次，预期的结果应该是什么？
>    ![](https://camo.githubusercontent.com/1b4b440d164f6a373f961fb208f538b6987804b3/68747470733a2f2f61736b2e71636c6f7564696d672e636f6d2f64726166742f313231353030342f347372303931653170642e706e67)
> 2. 如果要转移到自己的数据集，我需要更改代码的哪一部分？

请问你将模型转移到自己的数据集了吗,我也想在我的数据集上测试模型,不知道要怎么改变,如果你已经成功转移,可否教我一下,感激不尽
